druid/src/widget/align.rs

```diff
@@ -137,3 +140,14 @@ impl<T: Data> Widget<T> for Align<T> {
         self.child.paint_with_offset(ctx, data, env);
     }
 }
+
+fn log_size_warnings(size: Size) {
+    if size.width.is_infinite() {
+        println!("what");
```

Comment:
:exclamation:

Reply:
that's embarrassing! thank you for catching